### PR TITLE
[#174191947] Add new route for home page

### PR DESCRIPTION
**Pull request by Vadim-Kurnatovskiy**
**Stories:**

[Check whether we can have /categories page separately from /](https://www.pivotaltracker.com/story/show/174191947)
[landing page: header ](https://www.pivotaltracker.com/story/show/174274025)
[landing page: preview section](https://www.pivotaltracker.com/story/show/174148408)
[Landing page layout: communities cards](https://www.pivotaltracker.com/story/show/174005339)
[Landing page layout: footer](https://www.pivotaltracker.com/story/show/174148339)

[Design](https://projects.invisionapp.com/d/main#/console/20307299/425909330/preview)

**Desktop**

Снимок экрана 2020-08-13 в 17 31 57
Снимок экрана 2020-08-13 в 17 32 54

**Mobile**

Снимок экрана 2020-08-13 в 17 34 34
Снимок экрана 2020-08-13 в 17 35 06
Снимок экрана 2020-08-13 в 17 35 23

**Unauthorized**
Снимок экрана 2020-08-13 в 18 09 36
Снимок экрана 2020-08-13 в 17 44 58

### CODE REVIEW

File: app/controllers/communitarian/page_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Communitarian
+  class PageController < ::ApplicationController
+    requires_plugin Communitarian
+
+    def index
+      render json: { title: "Hello world!" }
```
Comment (ShaidullinArtur) [inline app/controllers/communitarian/page_controller.rb]: do we really need this action?)

Comment (IlgizGilmanov) [reply app/controllers/communitarian/page_controller.rb]: With the current implementation, most likely not needed. But what if we move the logic for getting categories here from the home route. Does this make sense and benefits?

Comment (IlgizGilmanov) [reply app/controllers/communitarian/page_controller.rb]: Removed this action

**Review by IlgizGilmanov — APPROVED** [review]

**Review by ShaidullinArtur — APPROVED** [review]

File: app/controllers/communitarian/page_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Communitarian
+  class PageController < ::ApplicationController
+    requires_plugin Communitarian
```
Comment (Vadim-Kurnatovskiy) [inline app/controllers/communitarian/page_controller.rb:5]: maybe use the empty method `def index; end`?

**Review by DmitryBarskov — APPROVED** [review]

**Pull request merged**